Samples/Tutorial Samples/YOLOv4ObjectDetection/YOLOv4ObjectDetection/Model.cs

```diff
@@ -4,10 +4,10 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
-using Windows.AI.MachineLearning;
```

Comment:
can you leave this in but commented out with a comment that says, "'Windows.AI.MachineLearning' refers to the libraries included in Windows, while 'Microsoft.AI.MachineLearning' refers to the externally available NuGet package." or something like that. what do you think? It would be good for people to know how to use either.

Reply:
Yea that makes sense.